gateway/modules/crud/crud.go

```diff
@@ -143,6 +146,15 @@ func (m *Module) SetConfig(project string, crud config.Crud) error {
 			v.Type = k
 		}
 
+		// check if connection string starts with secrets
+		s := strings.Split(v.Conn, ".")
```

Comment:
write a function for doing this
which will return secretName,secretkey,isSecretExists